functional-samples/ai.gemini-on-device-summarization/manifest.json

```
@@ -1,8 +1,12 @@
 {
-  "name": "Summarizer",
+  "name": "Summarization API sample",
   "version": "0.1",
   "manifest_version": 3,
-  "description": "Get a short summary of every webpage.",
+  "description": "Get a short summary of every webpage using Chrome's built-in summarization API.",
```

```suggestion
  "description": "Get a short summary of webpage content using Chrome's built-in summarization API.",
```

"Every" would be a lie, given the context window constraints.